lib/utilities/locationConstraintCheck.js

```diff
@@ -0,0 +1,35 @@
+import { errors } from 'arsenal';
+
+import config from '../Config';
+import { BackendInfo } from '../api/apiUtils/object/BackendInfo';
```

Comment:
I would move this locationConstraintCheck file to api/apiUtils/object as well.  It's used only on object operations, correct?  We've been trying to split our utilities so they are somewhat easy to find.